lib/mongoid/lazy_migration/document.rb

```diff
@@ -63,8 +63,7 @@ def try_lock_migration
     selector = atomic_selector.merge('migration_state' => { "$in" => [nil, 'pending'] })
     changes  = { "$set" => { 'migration_state' => 'processing' }}
     safety   = { :safe => true }
-
-    self.class.with(safety).where(selector).query.update(changes)['updatedExisting']
+    self.class.with(safety).where(selector).update(changes).successful?
```

Comment:
It's an alias...

```
     # Update the document attributes in the database.
      #
      # @example Update the document's attributes
      #   document.update(:title => "Sir")
      #
      # @param [ Hash ] attributes The attributes to update.
      #
      # @return [ true, false ] True if validation passed, false if not.
      #
      # @since 1.0.0
      def update(attributes = {})
        assign_attributes(attributes)
        save
      end
      alias :update_attributes :update
```